docs/includes/file-get-started-excel-angular.md

```diff
@@ -4,156 +4,40 @@ In this article, you'll walk you through the process of building an Excel add-in
 
 ## Prerequisites
 
-- Check whether you already have the [Angular CLI prerequisites](https://github.com/angular/angular-cli#prerequisites) and install any prerequistes that you are missing.
-
-- Install the [Angular CLI](https://github.com/angular/angular-cli) globally. 
-
-    ```bash
-    npm install -g @angular/cli
```

Comment:
Just to clarify for my own edification, does yo office install angular now?

Reply:
Yo Office installs all necessary dependencies for the chosen project type, so this step is no longer necessary.